app/javascript/utils/cookie_helper.js

```diff
@@ -0,0 +1,50 @@
+/**
+ *  @private
+ *  @summary Fetches and returns a cookie by cookie name
+ *  @param {string} name - Cookie name which the method should lookup
+ *  @returns {string} Cookie string in key=value format
+ *  @returns {null} if Cookie is not found
+ */
+function getCookie (name) {
+  const nameEQ = name + '='
+  const cookies = document.cookie.split(';')
+  for (let i = 0, len = cookies.length; i < len; i++) {
+    let cookie = cookies[i]
+    while (cookie.charAt(0) === ' ') {
+      cookie = cookie.substring(1, cookie.length)
```

Comment:
```suggestion
      cookie = cookie.substring(1, len)
```

Otherwise it was a bit pointless at the start.

Reply:
isn't `len = cookies.length` checking how many cookies there are, and `cookie.length` looking at the length of each separate cookie?